conf/config.yaml

```diff
@@ -15,6 +15,7 @@
 # limitations under the License.
 #
 apisix:
+  name: APISIX                   # APISIX gateway name
```

Comment:
as discuss in mailing list:https://lists.apache.org/thread.html/r2856157a47272100b97cf8c93e93933edfcfc9724477625e531ee1b0%40%3Cdev.apisix.apache.org%3E, we'd better not add name in config.yaml.

Reply:
close it first